App.tsx

```
@@ -56,6 +56,23 @@ const redactBreadcrumb = (breadcrumb: any) => {
   return breadcrumb;
 };
 
+// Benign/expected errors that carry no diagnostic value and should not be reported to
+// Sentry: normal user actions (wrong password, wrong PIN, cancelled HiveSigner signing)
+// and TanStack Query's internal cancellation signal.
+const IGNORED_ERROR_VALUES = ['auth.invalid_credentials', 'auth.invalid_username'];
+const IGNORED_ERROR_PATTERNS = [/check your PIN/i, /HiveSigner signing cancelled/i, /CancelledError/];
```

ESLint

[error] Replace `/check·your·PIN/i,·/HiveSigner·signing·cancelled/i,·/CancelledError/` with `⏎··/check·your·PIN/i,⏎··/HiveSigner·signing·cancelled/i,⏎··/CancelledError/,⏎`

(prettier/prettier)